_data/_samples.json

```diff
@@ -87,9 +79,12 @@
     {
       "title": "'Hello, blinky!'",
       "description": "A 'Hello, world' sample to control an LED from a breadboard",
-      "link": "samples/HelloBlinky.htm",
+      "link": "https://github.com/Microsoft/Windows-iotcore-samples/tree/develop/Samples/HelloBlinky",

```

Comment:
Links like this should all reference master, not develop.  It was develop before in ms-iot/samples.